package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@codesignal/meteor-protomongo",
+  "version": "0.0.3",
+  "description": "Extends meteor/mongo with handy async methods",
+  "main": "lib/mongo.js",
+  "author": "CodeSignal",
+  "scripts": {
+    "flow": "flow",
+    "build": "npm run build:clean && npm run build:lib",
+    "build:clean": "rimraf lib",
+    "build:lib": "babel src -d lib",
+    "build:flow": "flow-copy-source -v src lib"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-eslint": "^9.0.0",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-flow": "^6.23.0",
+    "babel-preset-minify": "^0.4.3",
+    "eslint": "^5.5.0",
+    "eslint-config-airbnb": "^17.1.0",
+    "eslint-plugin-flowtype": "^2.50.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-jsx-a11y": "^6.1.1",
+    "eslint-plugin-react": "^7.11.1",
+    "flow": "^0.2.3",
+    "flow-bin": "^0.80.0",
+    "flow-copy-source": "^2.0.2",
+    "rimraf": "^2.6.2"
+  },
+  "contributors": [
```

Comment:
I don't mean to step on your contributions (which are greatly appreciated!), but this `contributors` field really isn't necessary for a package managed by an organization, which is maintained/owned by the group collectively and not so much by independent, unaffiliated open source contributors